autoload/bm.vim

```diff
@@ -199,6 +223,12 @@ endfunction
 
 " Private {{{
 
+function! bm#compare_bm(bm1, bm2)
```

Comment:
The function is comparing by `sign_idx`, maybe `bm#compare_sign_idx` would be a more descriptive name? In fact in order to make it consistent with the existing behaviour of `bm#compare_lines` we could just pass in the `sign_idx`.